src/qentry.c

```diff
@@ -740,7 +740,7 @@ static bool _save(qentry_t *entry, const char *filepath)
     if (entry == NULL) return false;
 
     FILE *fd;
-    if ((fd = fopen(filepath, "w")) < 0) {
+    if ((fd = fopen(filepath, "w")) == NULL) {
```

Comment:
This looks good, thanks!
However a more concise way would be not to depend on the definition of `NULL` (see https://stackoverflow.com/a/3825704).
If you would feel like changing this to
```c
FILE *fd = fopen(filepath, "w");
if (!fd) {
```
I think that'd be cleaner. And doing the same for the following `_load()` function as well would be nice.